src/components/Class/index.js

```diff
@@ -119,6 +136,7 @@ function Class() {
               type="email"
               name="facilitator_email"
               id="facilitator_email"
+              pattern="[a-zA-Z0-9.-_]{1,}@[a-zA-Z.-]{2,}[.]{1}[a-zA-Z]{2,}"
```

Comment:
Yes, because the email type field is accepting komal@navgurkulorgt. It is throwing errors from the backend. by using the above pattern email type field will not accept komal@navgurkulorg text from an input.